Ginger/GingerCoreNET/RunLib/CLILib/CLIScriptFile.cs

```diff
@@ -57,14 +57,19 @@ public string CreateContent(Solution solution, RunsetExecutor runsetExecutor, CL
             return txt;
         }
 
-        public void Execute(RunsetExecutor runsetExecutor)
-        {                                    
-            var rc = CodeProcessor.ExecuteNew(mScriptFile);            
+        public void LoadGeneralConfigurations(string content, CLIHelper cliHelper)
+        {
+            mScriptFile = content;
         }
 
-        public void LoadContent(string content, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
+        public void LoadRunsetConfigurations(string content, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)